bio-qiime-pipeline/src/caching.py

```diff
@@ -0,0 +1,142 @@
+from typing import Optional
+
+import logging
+
+from coretex import CustomDataset, Experiment, SequenceDataset
+from coretex.utils.hash import hashCacheName
```

Comment:
why are we using absolute import here? can't this be shortened to `from coretex.utils import hashCacheName`?